WindowState/README.md

```diff
@@ -221,13 +221,13 @@ Returns whether a fold occludes content in the window.
 Based on the [occlusionType](https://developer.android.com/reference/androidx/window/layout/FoldingFeature#occlusionType()) field in [FoldingFeature](https://developer.android.com/reference/androidx/window/layout/FoldingFeature).
 
 ```kotlin
-val windowWidthDp: Dp = 0.dp
+val windowWidthDp: Dp = 1.dp
```

Comment:
This is because the JWM WindowSizeClass implementation requires a size greater than zero:
```
internal fun compute(dpWidth: Float): WindowWidthSizeClass {
            require(dpWidth > 0) { "Width must be positive, received $dpWidth" }
            return when {
                dpWidth < 600 -> COMPACT
                dpWidth < 840 -> MEDIUM
                else -> EXPANDED
            }
        }
```
I wasn't sure if it would be better to change the default value or remove the default values - any preferences?

Reply:
Oh I see, got it! It's fine as you have changed, all good. Thanks!